install-dev/data/db_structure.sql

```diff
@@ -910,7 +910,7 @@ CREATE TABLE IF NOT EXISTS `PREFIX_gender_lang` (
 /* Group special price rules */
 CREATE TABLE `PREFIX_group` (
   `id_group` int(10) unsigned NOT NULL auto_increment,
-  `reduction` decimal(17, 2) NOT NULL DEFAULT '0.00',
+  `reduction` decimal(5, 2) NOT NULL DEFAULT '0.00',
```

Comment:
17 => 5 ?

Reply:
Yes, because `reduction` is a percentage and in the you can't add a reduction above 100% so there is no need to have 15 digits for the integer part.

Reply:
:ok_hand:

Reply:
But it's a BC ! Ping @jolelievre

Reply:
Indeed strictly speaking it's a BC and we will have to document and inform about it But in reality it's like the other changed types, although the range is less wide it shouldn't contain such big values anyway So we're safe

Reply:
:ok_hand:

Reply:
Are you sure reduction is only a percentage? :trollface: 
FYI https://github.com/PrestaShop/PrestaShop/blob/develop/classes/GroupReduction.php#L41